recipes/depot_tools/all/conanfile.py

```diff
@@ -30,63 +39,68 @@ def _dereference_symlinks(self):
         if self.settings.os != "Windows":
             return
 
-        for root, dirs, files in os.walk(self._source_subfolder):
+        for root, dirs, files in os.walk(self.source_folder):
             symlinks = [os.path.join(root, f) for f in files if os.path.islink(os.path.join(root, f))]
             for symlink in symlinks:
                 dest = os.readlink(symlink)
                 os.remove(symlink)
                 shutil.copy(os.path.join(root, dest), symlink, follow_symlinks=False)
                 self.output.info("Replaced symlink '%s' with its destination file '%s'" % (symlink, dest))
 
-    def source(self):
-        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder)
+    def build(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder)
```

Comment:
Why is this part of the build method instead of the source one? It seems like the sources are configuration independent

Reply:
Mostly just following the example in the docs:
https://github.com/conan-io/conan-center-index/blob/ff8b4aaf4ea7e4af4306ab956ce31f1ff026308c/docs/package_templates/prebuilt_tool_package/all/conanfile.py#L40-L46
As far as I can tell it serves the same purpose as `no_copy_source=True`.